react/react.py

```diff
@@ -96,7 +98,7 @@ def to_json(data):
         :return: Received data in json format
         :rtype: str
         """
-        if isinstance(data, str):
+        if isinstance(data, six.string_types):
```

Comment:
This should be `six.text_type` to keep the py3 behaviour.

But I dunno what this is doing here, it must be a leftover from some refactory. I will remove it at some point (not now coz it breaks backward-compatibility).